windows/installer-source/KolibriSetupScript.iss

```diff
@@ -31,11 +31,19 @@ UsePreviousAppDir=yes
 ChangesEnvironment=yes
 
 [Languages]
-Name: "English"; MessagesFile: "compiler:default.isl"
+Name: "english"; MessagesFile: "compiler:Default.isl"
+Name: "spanish"; MessagesFile: "compiler:Languages\Spanish.isl"

```

Comment:
curious why `english` and `spanish` are lowercase but the other languages are uppercase?